spec/spec_helper.cr

```diff
@@ -1,29 +1,13 @@
 require "spec"
 require "../src/garnet_spec"
 
-module GarnetSpec
-  DRIVER = :chrome
-  PATH   = "/usr/local/bin/chromedriver"
+class FakeHandler
+  def call(context)
+    response = { headers: context.request.headers.to_h }.to_json
+    context.response.write(response.to_slice)
+  end
+end
 
-  # Not all server implementations will support every WebDriver feature.
-  # Therefore, the client and server should use JSON objects with the properties
-  # listed below when describing which features a session supports.
-  capabilities = {
```

Comment:
There is no need to provide these options in spec helper since there are defaulted anyways.